esp/src/src-react/components/Metrics.tsx

```diff
@@ -67,8 +67,12 @@ class TableEx extends Table {
                 return row[field]?.indexOf && row[field]?.indexOf(filter.substring(colonIdx + 1)) >= 0;
             }
             for (const key in row) {
-                if (row[key]?.indexOf && row[key]?.indexOf(filter) >= 0) {
-                    return true;
+                if (row[key]?.indexOf) {
+                    const value = (!matchCase && row[key]?.toLowerCase) ? row[key]?.toLowerCase() : row[key];
+                    const filterValue = (!matchCase && row[key]?.toLowerCase) ? filter.toLowerCase() : filter;
```

Comment:
should this be `!matchCase && filter?.toLowerCase`?

Reply:
No this is correct - the idea is to only lower the filterValue (which is always a string) if the value had been lowered

Reply:
Updated logic